aws-lc-rs/src/aead/aead_ctx.rs

```diff
@@ -71,24 +153,48 @@ impl AeadCtx {
                 aead,
                 key_bytes.as_ptr().cast(),
                 key_bytes.len(),
-                TAG_LEN,
+                tag_len,
                 null_mut(),
             ) {
                 return Err(Unspecified);
             }
             Ok(aead_ctx.assume_init())
         }
     }
+
+    pub(crate) fn as_mut_ptr(&mut self) -> &mut EVP_AEAD_CTX {
```

Comment:
It would be more idiomatic to implement `AsMut<EVP_AEAD_CTX>` and `AsRef<EVP_AEAD_CTX>`.

Reply:
Those would be public impls, though

Reply:
It would be implemented on `AeadCtx` which is only `pub(crate)` so it should be ok.(?)

Reply:
Ah this is my mad I had originally meant this to be similar to DigestCtx where these just return the raw pointers. Will take a look again.